packages/components/src/Modal/GenericModal/subcomponents/ModalContext.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from "react"
+
+export type ModalContextType = {
+  labelledByID: string
+  describedByID: string
+}
+
+export const ModalContext = createContext<ModalContextType>({
```

Comment:
I'm pretty sure we tend to put contexts into a `context` directory instead of `subcomponents`.